backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -1117,20 +1117,97 @@ template <typename Torus> struct int_sum_ciphertexts_vec_memory {
   }
 };
 
+template <typename Torus> struct int_zero_out_if_buffer {
```

Comment:
Didn't we alreadyy have one like this? :thinking:

Reply:
Yes I just moved it a bit above.